tests/common/type_coverage.h

```diff
@@ -2,7 +2,7 @@
 //
 //  SYCL 2020 Conformance Test Suite
 //
-// Provide common functions for type coverage
+//  Provide common functions for type coverage
```

Comment:
```suggestion
// Provide common functions for type coverage
```

Reply:
I think we should leave two spaces for consistency with the rest codebase.
Few examples of sources where two spaces used in the file header:
- [First](https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/tests/image/image_api.cpp)
- [Second](https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/tests/common/macros.h)
- [Third](https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-2020/tests/group/group_api.cpp)